docs/platform/webcontainers/browser-support.md

```diff
@@ -26,7 +26,7 @@ WebContainers are supported in all recent desktop browsers (fully in Chrome and
 ### Mobile browsers
 
 - Android: beta support for Chrome, Chromium-based browsers and Firefox. Depending on your device, large projects may run into memory limitations.
-- iOS and iPadOS: not supported yet. WebContainers can theoretically work on iOS 16.4 and later, but memory limitations for web apps in Safari for iOS and iPadOS are currently too strict to run most WebContainers-based projects.
+- iOS and iPadOS: beta support since Safari 16.4. Large projects may run into memory limitations because the memory usage for a web page on a mobile device is more constrained. In this case, we recommend switching to a Desktop browser that is less constrained and has more memory.
```

Comment:
On iOS, I think it might be more common to refer to the iOS version than to the Safari version. Not sure it's easy to check the Safari version you're using, for end users.

So maybe it should be `beta support for Safari since iOS 16.4`?

Reply:
Yep we can definitely rephrase. Open to anything here 👍

Reply:
Updated. How does that sound now?